src/views/SelectProductCSVUpload.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-back-button slot="start" default-href="/" />
+        <ion-title>{{ $t("Upload CSV") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content :fullscreen="true">
+      <div class="upload-csv">
```

Comment:
use semantic tag here and update css selector

Reply:
Done